ReactSkia/BUILD.gn

```diff
@@ -5,11 +5,18 @@ declare_args() {
   rns_enable_api_profiling = false
 }
 
+import("//build/config/gclient_args.gni")
 config("ReactSkia_config") {
   defines = []
   if(rns_enable_api_profiling) {
     defines += ["RNS_ENABLE_API_PERF"]
   }
+  if( react_native_target == "default") {
+     print(" ** React Native Target :  Default **")
```

Comment:
Do we still need these prints?

Reply:
The print was added for test purpose. Will remove the print.